doc/api/cli.md

```diff
@@ -2323,6 +2309,22 @@ added:
 Configures the test runner to exit the process once all known tests have
 finished executing even if the event loop would otherwise remain active.
 
+### `--test-isolation=mode`
+
+<!-- YAML
+added: v22.8.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/56298
+    description: Test isolation configuration is now stable.
```

Comment:
I feel like it would make more sense to document the change of stability on `test.md`, and here document only the renaming of the flag.
```suggestion
    description: This flag was renamed from `--experimental-test-isolation` to
                 `--test-isolation`.
```

Reply:
OK, I've merged your suggestion in.